packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx

```diff
@@ -271,105 +274,46 @@ const ProofOfAddressForm = observer(
         } else {
             form_initial_values.address_state = '';
         }
-        const setOffset = (status: { msg: string }) => {
-            const mobile_scroll_offset = status?.msg ? '200px' : '154px';
-            return !isDesktop && !is_for_cfd_modal ? mobile_scroll_offset : '80px';
+        const setOffset = (status: { msg: string }): string => {
```

Comment:
is it really necessary to add 'string' as return type? I think TS understands it from the conditions

Reply:
We can rely on type inference